File: C/Colleen/Colleen.c
#include <stdio.h>
void func(char *str) {
	printf(str, 10, 34, str) ;
}
// A comment
int main() {
	// A cool comment
	char *str = "#include <stdio.h>%1$cvoid func(char *str) {%1$c	printf(str, 10, 34, str) ;%1$c}%1$c// A comment%1$cint main() {%1$c	// A cool comment%1$c	char *str = %2$c%3$s%2$c ;%1$c	func(str) ;%1$c}" ;
	func(str) ;
}

File: Rust/Colleen/Colleen.rs
fn func() {
	let str: &str = "fn func() {{{0}	let str: &str = {1}{3}{1};{0}	print!({1}{3}{1}, '{2}n', '{1}', '{2}{2}', str);{0}}}{0}// A comment{0}fn main() {{{0}	// A cool comment{0}	func();{0}}}";
	print!("fn func() {{{0}	let str: &str = {1}{3}{1};{0}	print!({1}{3}{1}, '{2}n', '{1}', '{2}{2}', str);{0}}}{0}// A comment{0}fn main() {{{0}	// A cool comment{0}	func();{0}}}", '\n', '"', '\\', str);
}
// A comment
fn main() {
	// A cool comment
	func();
}